DotNet.DocsTools/Utility/ContentScrubber.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Text.RegularExpressions;
+
+namespace DotNet.DocsTools.Utility;
+
+public static partial class ContentScrubber
+{
+    private const string Replacement = "<i>Image link removed to protect against security vulnerability.</i>";
+
+    /// <summary>
+    /// Remove false security vulnerabilities from the content.
+    /// </summary>
+    /// <param name="content">The source HTML content</param>
+    /// <returns>The content with any tokens removed.</returns>
+    /// <remarks>
+    /// Links to images on GitHub contain a JWT token that
+    /// creates a false positive in our credential scan.
+    /// The token is a secret that expires. As a result, it won't
+    /// work as a link, but also puts triggers a security warning.
+    /// We need to scrub these links from the content.
+    /// Replace the link with a placeholder that says
+    /// "Image link removed to protect against security vulnerability."
```

Comment:
But it really isn't a vulnerability, as there's no real "secret" that needs to be protected. How about "Image link removed. See the original GitHub issue to see the image."